intro-to-python-testing.md

```diff
@@ -9,6 +9,12 @@
 
 DataMade uses the `pytest` framework for testing Python. `pytest` auto-discovers all test modules by recursing into test directories and searching for `test_*.py` and `*_test.py` files. This mode of test discovery allows for some flexibility in directory-structure design: the tests can reside within nested app directories or in a stand-alone directory at the root of the main repo. At DataMade, we prefer to the latter structure. Here’s how to get started.
 
+To install pytest, run:
+
+```bash
+pip install -U pytest
```

Comment:
we use virtual environments for our projects, so we don't need to specify -U (for user installation, as opposed to global installation) here.